client/src/components/FoodSeeker/SearchResults/StakeholderPreview/StakeholderPreview.js

```diff
@@ -251,24 +292,210 @@ const StakeholderPreview = ({ stakeholder, onSelect }) => {
+                      >
+                        {`Closing in ${minutesToClosing} minutes`}
+                      </Typography>
+                    </Box>
+                  )}
+
+                  {isAlmostOpenFlag && (
+                    <Box
+                      textAlign="left"
+                      marginTop="4px"
+                      sx={{ flexBasis: "100%" }}
+                    >
+                      <Typography
+                        variant="body2"
+                        component="p"
+                        fontWeight="bold"
+                        color={success}
+                        key={stakeholder.id}
+                      >
+                        {`Opening in ${minutesToOpening} minutes`}
+                      </Typography>
+                    </Box>
+                  )}
+                </AccordionSummary>
+                <AccordionDetails>
+                  {stakeholder.hours ? (
+                    <Stack>
+                      <Divider sx={{ margin: "8px 0px 4px" }} />
+                      {stakeholder.hours && stakeholder.hours.length > 0 ? (
+                        stakeholder.hours.sort(hoursSort).map((hour, index) => (
+                          <Stack
+                            direction="row"
+                            justifyContent="space-between"
+                            key={JSON.stringify(hour)}
+                            marginTop="4px"
+                            sx={{
+                              ".MuiTypography-root": {
+                                fontWeight:
+                                  index === 0 ||
+                                  isCurrentDayAndWeek(tenantTimeZone, hour)
+                                    ? "bold"
+                                    : "",
+                              },
+                            }}
+                          >
+                            <Typography>
+                              {hour.week_of_month === -1
+                                ? "Last " + hour.day_of_week
+                                : hour.week_of_month === 1
+                                ? "1st " + hour.day_of_week
+                                : hour.week_of_month === 2
+                                ? "2nd " + hour.day_of_week
+                                : hour.week_of_month === 3
+                                ? "3rd " + hour.day_of_week
+                                : hour.week_of_month === 4
+                                ? "4th " + hour.day_of_week
+                                : hour.day_of_week}
+                            </Typography>
+                            <Typography>
+                              {standardTime(hour.open)} -{" "}
+                              {standardTime(hour.close)}
+                            </Typography>
+                          </Stack>
+                        ))
+                      ) : (
+                        <Typography>No hours on record</Typography>
```

Comment:
Since this is not used, we can remove this element from the `AccordianDetails` element.